src/app/tests/suites/certification/Test_TC_MOD_1_1.yaml

```diff
@@ -77,27 +99,58 @@ tests:
               contains: [4]
 
     - label:
-          "TH reads the optional attribute(OnMode) in AttributeList from the DUT"
-      PICS: MOD.S.A0005
+          "Read the Feature dependent(MOD.S.F00 - DEPONOFF) and optional attribute(OnMode) in AttributeList from the DUT"
+      PICS: MOD.S.A0005 && MOD.S.F00
       command: "readAttribute"
       attribute: "AttributeList"
       response:
           constraints:
               type: list
               contains: [5]
 
-    - label: "Read the global attribute: AcceptedCommandList"
+    - label: "Read the global attribute AcceptedCommandList. Check if it contains the Feature dependent (MOD.S.F02 - DEPCTM) command with id 0x0 (ChangeToMode)"
+      PICS: MOD.S.Afff9 && !MOD.S.F02
```

Comment:
Why is this conditional on MOD.S.Afff9?  This is a required attribute.

Reply:
@bzbarsky-apple Your question makes sense to me. 

@ReneJosefsen Is there any reason we need a PICS code for the required attributes, in these tests? The Window Covering or the Pump Config & Control tests, for example, also do this.

Reply:
@bzbarsky-apple I brought this up earlier today with the CSG Test Plan TT. The answer I got was that it's useful to make parts of the tests conditional so you can disable them during development, if needed.

Reply:
OK.  I also talked with the CSG folks and they said that when devices come in for cert they require that the "must be supported" PICS are listed as supported, so things should be ok.

Reply:
Hmm, I checked the PCC test and is does include this and seems to be added in https://github.com/project-chip/connectedhomeip/pull/26627, but honestly I think this is bug/issue since the PICS XML does not include PICS for global elements (https://github.com/CHIP-Specifications/chip-test-plans/pull/2611) 🤔

Reply:
It looks like the CI does define PICS values for these global attributes:

PCC.S.Afffd=1
PCC.S.Afffc=1
PCC.S.Afffb=1
PCC.S.Afffa=1
PCC.S.Afff9=1
PCC.S.Afff8=1

https://github.com/project-chip/connectedhomeip/blob/9f439889aed5581b53ea79d5975a4b78e4510998/src/app/tests/suites/certification/ci-pics-values#L908

Reply:
I think I have a somewhat understanding of why this was added. The latest template for the 1.1 test cases does include these global attribute PICS, which I in hindsight, properly should not have added. I will follow up on this and figure out if we should remove these from the test plans (and YAML scripts) or if the PICS XML should be updated to include these as well.